x/crisis/internal/types/types.proto

```diff
@@ -7,14 +7,7 @@ import "third_party/proto/gogoproto/gogo.proto";
 
 // MsgVerifyInvariant - message struct to verify a particular invariance
 message MsgVerifyInvariant {
-  bytes sender = 1 [
-    (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress",
-    (gogoproto.moretags) = "yaml:\"sender\""
```

Comment:
remove yaml here?

Reply:
Yes -- there is no need to define the yaml tag if the field is just one word.